src/System.ComponentModel.Composition/tests/System/ComponentModel/Composition/Hosting/DirectoryCatalogTests.cs

```diff
@@ -343,6 +343,19 @@ public void LoadedFiles_ContainsMultipleDllsAndSomeNonDll_ShouldOnlyContainDlls(
                     cat.LoadedFiles);
         }
 
+        [Fact]
+        public void LoadedFiles_NonStaticallyReferencedAssembly()
+        {
```

Comment:
@safern i added a test

Reply:
Could you add a similar test for AssemblyCatalog now that you moved the code there?